hot-reload-agent/src/main/kotlin/org/jetbrains/compose/reload/agent/resources.kt

```diff
@@ -55,14 +57,15 @@ private fun cleanResourceCache(classId: ClassId, cleanMethod: MethodId, resource
 }
 
 internal fun Context.cleanResourceCacheIfNecessary() {
-    if (hasChangedResources()) {
-        cleanResourceCache(Ids.ImageResourcesKt.classId, Ids.ImageResourcesKt.dropImageCache, "Images")
-        cleanResourceCache(
-            Ids.StringResourcesUtilsKt.classId,
-            Ids.StringResourcesUtilsKt.dropStringItemsCache,
-            "Strings"
-        )
-    }
+    if (!HotReloadEnvironment.composeReloadResourcesEnabled) return
```

Comment:
Cleaning resource caches is quite safe, I don't think we need to put it under feature flag.